src/airflow/airqo_etl_utils/schema/measurements.json

```diff
@@ -0,0 +1,156 @@
+[
+    {
```

Comment:
What is the main difference between `measurements` and `datawarehouse` schemas? Could you provide some context?  @NoahNsimbe

Reply:
measurements are pm2_5, pm10 etc...datawarehouse has site details and weather data in addition to the measurements. You can compare the schemas below @Baalmart 

**Datawarehouse**
https://github.com/airqo-platform/AirQo-api/blob/PLAT-1040-stream-data-into-the-datawarehouse/src/airflow/airqo_etl_utils/schema/data_warehouse.json

**Measurements (Device Readings)**
https://github.com/airqo-platform/AirQo-api/blob/PLAT-1040-stream-data-into-the-datawarehouse/src/airflow/airqo_etl_utils/schema/measurements.json

**Weather data**
https://github.com/airqo-platform/AirQo-api/blob/PLAT-1040-stream-data-into-the-datawarehouse/src/airflow/airqo_etl_utils/schema/weather_data.json